src/local.rs.orig

```diff
@@ -0,0 +1,694 @@
+// Copyright (C) 2024 MaidSafe.net limited.
```

Comment:
Can we remove this file if it is not required? We can always revert back a commit right?